lib/dry/web/parser.rb

```diff
@@ -0,0 +1,54 @@
+module Dry
+  module Web
+    class Parser
```

Comment:
I feel like this should have a slightly more specific name, perhaps `Dry::Web::Settings::FileParser`?